package.json

```
@@ -5,12 +5,12 @@
   "main": "index.js",
   "dependencies": {
     "inherits": "~2.0.4",
-    "readable-stream": "^3.5.0"
+    "readable-stream": "^4.3.0"
```

```suggestion
    "readable-stream": "^3.6.2"
```
This is a breaking change and addressed in #29 so maybe it makes sense to keep this bump to latest non-breaking?